Sources/Classes/RSClient.m

```diff
@@ -38,6 +38,9 @@ + (instancetype)getInstance:(NSString *)writeKey config:(RSConfig*)config {
 }
 
 + (instancetype)initiate:(NSString *)writeKey config:(RSConfig * __nullable)config options:(RSOption * __nullable)options {
+    if ([writeKey length] == 0) {
```

Comment:
Same as in Android, can we update the log message to say SDK initialization is aborted as no write key is provided and return back immediately ?

Reply:
Initialization is not aborted & will not aborted. We are not aborting initialisation.